src/aposteriori/data_prep/create_frame_data_set.py

```diff
@@ -937,6 +1261,9 @@ def make_frame_dataset(
         If True, the user will be prompted to start creating the dataset.
     encode_cb: bool
         Whether to encode the Cb at an average position in the frame.
+    voxels_as_gaussian: bool
+        Whether to encode voxels as gaussians.
```

Comment:
As this is the top level function, it is the function that people are most likely to interact with, so it'd be good to have a fully description here, maybe with the citation or explaining the problem it attempts to solve.